server/artifacts/artifact_server.go

```diff
@@ -95,7 +95,7 @@ func (a *ArtifactServer) GetArtifactFile(w http.ResponseWriter, r *http.Request)
 	direction := requestPath[directionIndex]
 	artifactName := requestPath[artifactNameIndex]
 
-	if direction != "outputs" { // for now we just handle output artifacts
+	if direction != "outputs" && direction != "inputs" { // for now we handle output and input artifacts
```

Comment:
Update the comments above the function.

Reply:
Probably a good idea to introduce a string enum at this stage, just makes a it a bit more structured

```go
type Direction string

const (
	Outputs Direction = "outputs"
	Inputs Direction = "inputs"
)
```

Reply:
okay, I have change it.